docs/BUILD.md

```diff
@@ -26,6 +26,28 @@ To build a JAR that can run stand-alone without any additional classes on the cl
 $ ./gradlew shadowJar
 ```
 
+## Packaging as Installable Application
+
+To build an installable application package appropriate for your operating system
+(e.g. Windows, Mac OS, Linux), first make sure you have a recent version of the
+JDK installed (ver >= 15) that includes `jlink` and `jpackage`.  If you intend
+to redistribute the built application publicly, make sure it's an OpenJDK
+distribution (likely a recent build linked from https://openjdk.java.net/install/,
+ok if it's built by Oracle) and *NOT* an Oracle *commercial* JDK, where license
+and redistribution terms are murkier.
```

Comment:
I'd suggest mentioning the WiX requirement here as well.

Also, on two machines (Windows 10 Home and Windows 10 Pro for Workstations) when trying to install the WiX executable from https://github.com/wixtoolset/wix3/releases/tag/wix3112rtm, I got this message:

![image](https://user-images.githubusercontent.com/928045/167674661-6f03df1f-9f06-480c-8a19-a00f3471b27e.png)

And then I tracked this down for instructions to enable the .NET framework:
https://stackoverflow.com/a/57820594/937715

We may want to include brief instructions here for doing the above workflow.

I was able to install the MSI installer on my Windows 10 Home machine WITHOUT having to enable .NET framework 3.5.1, so it looks like this is only a requirement to build the package, not for end-users installing the package (comments in the above SO post imply that it's required for end users as well, but it wasn't for me).

FYI, for future reference, the packaged MSI is 36.6 MB and the installed application size (as listed by Windows) is 48.9 MB.

Reply:
Done in c3588359a1727c9efcc573388911c1e44cd85f6a.